docs/source/write_first_app.rst

```diff
@@ -179,7 +179,9 @@ Let's run the application and then step through each of the interactions with th
 ``asset-transfer-basic/application-gateway-typescript`` directory, run the following command:
 
 .. code-block:: bash
-
+  
+  npm run build
```

Comment:
The `npm install` command in the previous step already runs the TypeScript build. There is no need to explicitly run the build again, unless you are modifying the application code, in which case you can simply re-run `npm install`